Objects/typeobject.c

```diff
@@ -625,7 +625,7 @@ type_get_bases(PyTypeObject *type, void *context)
 
 static PyTypeObject *best_base(PyObject *);
 static int mro_internal(PyTypeObject *, PyObject **);
-static int type_is_subtype_base_chain(PyTypeObject *, PyTypeObject *);
+static int type_is_subtype_base_chain(const PyTypeObject *, const PyTypeObject *);
```

Comment:
Please do not add `const` to `PyObject` pointers (and `PyTypeObject` is a subtype of `PyObject`). They are inherently mutable. If the particular function does not mutate them in the particular Python version, it is just an implementation detail.

Reply:
Will do.  I know we've discussed before, but this was an old branch.   Sorry.  I'll resubmit.